test/StakeManager.js

```diff
@@ -566,6 +566,12 @@ describe('StakeManager', function () {
       const tx = stakeManager.connect(signers[1]).setDelegationAcceptance('true');
       await assertRevert(tx, 'comission not set');
     });
+
+    it('Staker should not be able to decreaseCommission if comission is zero', async function () {
```

Comment:
Hey, this revert doesn't indicate that if the old commission is zero, staker should not be able to decrease it,
It indicates once the commission is set, it cant be decreased to 0 
right?

If so, please update this title